vembrane/representations.py

```diff
@@ -253,7 +261,7 @@ def _get_alt(self) -> str:
         return alts[0]
 
     def _get_qual(self) -> float:
-        value = type_info(self.record.qual)
+        value = self.record.qual or NA
```

Comment:
A score of `0` could be -- although semantically questionable -- a valid input, right?

Reply:
You're right; I just forgot to adapt this one.
```suggestion
        value = NA if self.record.qual is None else self.record.qual
```

Reply:
This just came to mind: Does `record.qual` come directly from Pysam as a `float`? If so, do we need or want to handle (=convert to `NA`)`float('nan')`?

Reply:
```suggestion
        value = NA if (self.record.qual is None or isnan(self.record.qual)) else self.record.qual
```
needs an `from math import isnan` up top.
But it we could also leave it as `float('nan')` -- I haven't thought this one through ;).

Reply:
Yes, pysam has `QUAL` as float (I think). However, since `QUAL` may also be missing (i.e. `.` dot), `nan` and "missing" are two different things. So I'd rather not have `nan` be `NA`, only `.` is `NA`.

Reply:
SGTM!